word_guess.rb

```diff
@@ -0,0 +1,213 @@
+# Technical
+#
+# You should be able to play your game by running it using ruby from the terminal.
+# Use Ruby class objects to contain your game logic. You may be able to implement your solution using just one class, but consider how isolating related functionality into separate classes may increase your code's readability and maintainability.
+
+#Create an array of possible words for the player to guess. Use sample to get one word from array.
+# mystery_words = %w(get red kitty nose)
+#
+# #Print ASCII art and mystery word.
+# puts frogarray[0]
+# puts mystery_words.sample
+
+#Ask player to guess a letter.
+#Capture the guess and push to array "letters you've guessed"
+#Checks whether letter is in mystery word.
+
+#If true:
+#Print "Got it right!"
+#Print mystery word with correct guessed letter inserted.
+# variable current guess. When == mystery, they win!
+#Loop back to Guess.
+
+#Else/if false, print "Wrong!" and remove one section of ASCII art.
+#Print "Try again!"
+#Loop back to Guess.
+
+# a method that will print an updated image to the string. that informs of the amount left
+# store the art in an array which will be indexed [0].....ect
+# syntax: print pie array with index[1]
+
+#def froggy
+#end
+
+# #
+#      @ . . @
+#     (-------)
+#   (   >___<   )
+#   ^^  ~   ~   ^^
+#   \\\\\\\///////
+# #
+# #   @ . . @
+#  (-------)
+# (   >___<   )
+#
+#
+#
+#
+# ^^  ~   ~   ^^
+# \\\\\\\///////
+#
+#     @ . . @
+#    (-------)
+#   (   >___<   )
+#  ^^  ~   ~   ^^
+#  \\\\\\\///////
+
+#Reprint the ASCII image, the guessed letters the status of the mystery word
+# (create a method)
+
+#When word mystery word is complete
+# puts the correct word
+# print ASCII art
+# puts "you won!"
+# exit
+
+#When ASCII art is gone!
+# puts "you lost"
+# exit
+
+
+class WordGuess
+
+   frog1 = "       @ . . @
+      (-------)
+    (   >___<   )
+    ^^  ~   ~   ^^ ".colorize(:green) +
+    "\n   //////\\\\\\\///////
+    ~~~~~~~~~~~~~~~~
+    ~~~~~~~~~~~~~~~~ ".colorize(:blue)
+
+  frog2 = "       @ . . @
+      (-------)
+    (   >___<   )
+    ^^  ~   ~   ^^ ".colorize(:green) +
+      "\n   ~~~~~~~~~~~~~~
+      ~~~~~~~~~~~~~~
+      ~~~~~~~~~~~~~~ ".colorize(:blue)
+
+  frog3 = "       @ . . @
+      (-------)
+    (   >___<   ) ".colorize(:green) +
+    "\n   ~~~~~~~~~~~~~~
+    ~~~~~~~~~~~~~~
+    ~~~~~~~~~~~~~~
+    ~~~~~~~~~~~~~~".colorize(:blue)
+
+  frog4 = "       @ . . @
+      (-------)".colorize(:green) +
+      "\n   ~~~~~~~~~~~~~~
+      ~~~~~~~~~~~~~~
+      ~~~~~~~~~~~~~~
+      ~~~~~~~~~~~~~~
+      ~~~~~~~~~~~~~~ ".colorize(:blue)
+
+  frog5 = "       @ . . @".colorize(:green) +
+  "\n   ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~ ".colorize(:blue)
+
+  frog6 =  "  ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~ ".colorize(:blue)
+
+  frogarray = []
+  frogarray.push(frog1, frog2, frog3, frog4, frog5, frog6)
```

Comment:
If you initialize the `frogarray` before defining each string, you will not need to push each string in. You could put the strings in the array yourself:
`frog[0] = "..."`